static/js/admin.js

```diff
@@ -1295,117 +1301,6 @@ function remove_question(question_name) {
     });
 }
 
-// Called when the "Write" button is clicked.
-function display_write() {
-    var template = document.getElementById('template');
-    var questiontype = template.options[template.selectedIndex].value;
-    var obj = new XMLHttpRequest();
-    obj.open('POST', '/runestone/admin/gettemplate/' + questiontype, true);
-    obj.send();
-    obj.onreadystatechange = function () {
-        if (obj.readyState == 4 && obj.status == 200) {
-            var returns = JSON.parse(obj.responseText);
-            tplate = returns['template'];
-            $("#qcode").text(tplate);
-        }
-        $.each(returns['chapters'], function (i, item) {
-            $('#qchapter').append($('<option>', {
-                value: item,
-                text: item
-            }));
-        });
-    };
-
-    var hiddenwrite = document.getElementById('hiddenwrite');
-    hiddenwrite.style.visibility = 'visible';
-}
-
-
-// Called when the "Done" button of the "Write" dialog is clicked.
-function create_question(formdata) {
```

Comment:
This seems a bit overzealous.  We still want people to be able to write their own questions, preview them, and have them added to the DB.